tests/sqlparser_common.rs

```diff
@@ -106,6 +106,9 @@ fn parse_insert_values() {
     }
 
     verified_stmt("INSERT INTO customer WITH foo AS (SELECT 1) SELECT * FROM foo UNION VALUES (1)");
+
+    // allow parenthesis query after insert into
+    verified_stmt("INSERT INTO tbla (cola) (SELECT cola FROM tblb)");
```

Comment:
What does this syntax mean? Is it the same as 

```sql
INSERT INTO tbla (cola) SELECT cola FROM tblb
```

(no parens?)

Reply:
I believe so, well at the least, running an explain statement in Athena (Trino) yields no difference between

`INSERT INTO tbla (cola) SELECT cola FROM tblb`
and
`INSERT INTO tbla (cola) (SELECT cola FROM tblb)`

And registers both as valid queries.  From what I recall, I think you can generally wrap any select statements in quotes